test-suite/src/test/scala/org/scalajs/testsuite/javalib/IntegerTest.scala

```diff
@@ -193,5 +193,28 @@ object IntegerTest extends JasmineTest {
       expect(Integer.toString(-2147483648, 2)).toEqual("-10000000000000000000000000000000")
       expect(Integer.toString(-2147483648, 10)).toEqual("-2147483648")
     }
+
+    it("should provide `toString`") {
```

Comment:
This test is redundant as this test was split into the two above - with/without radix. Will remove it.